test/api.js

```diff
@@ -307,12 +307,23 @@ function generateTests(prefix, apiCreator) {
 			});
 	});
 
-	test(prefix + 'change process.cwd() to a test\'s directory', function (t) {
+	test(prefix + 'run from package.json folder by default', function (t) {
```

Comment:
This test doesn't do what you think it does.

The `apiCreator` executor doesn't actually set the `pkgDir` option when it creates the `api`. This means that the child process is forked with an `undefined` `cwd` option, so it ends up using `process.cwd()`. Because we run tests from the project root that happens to be the expected directory, but that's just luck.

Instead let's find the correct `pkgDir` value in the test itself, and provide it when creating the API [on this line](https://github.com/alathon/ava/blob/2a65c1295588ebdf36dcc833484c34f3ed956add/test/api.js#L20) and [this one](https://github.com/alathon/ava/blob/2a65c1295588ebdf36dcc833484c34f3ed956add/test/api.js#L65).

The next test is good (since it tests a non-default value). Just make sure not to accidentally break it with the aforementioned changes.